plugin.cpp

```diff
@@ -54,16 +54,19 @@
 #define xstr(s) str(s)
 #define str(s) #s
 
-#define DEF_RULE_CFG_VALUE  "{\"asset\":{\"description\":\"The asset name for which notifications will be generated.\",\"name\":\"modbus\"},\"datapoints\":[{\"type\":\"float\",\"name\":\"humidity\"},{\"type\":\"float\",\"name\":\"temperature\"}],\"expression\":{\"description\":\"The expression to evaluate\",\"name\":\"Expression\",\"type\":\"string\",\"value\":\"if( ((humidity > 50)), 1, 0)\"}}"
-
 #define RULE_DEFAULT_CONFIG \
 			"\"description\": { " \
```

Comment:
Whilst making this change could you please also convert to use the easier to read QUOTE macro method of defining the JSON.